aws/resource_aws_securityhub_account.go

```diff
@@ -58,10 +62,30 @@ func resourceAwsSecurityHubAccountDelete(d *schema.ResourceData, meta interface{
 	conn := meta.(*AWSClient).securityhubconn
 	log.Print("[DEBUG] Disabling Security Hub for account")
 
-	_, err := conn.DisableSecurityHub(&securityhub.DisableSecurityHubInput{})
+	err := resource.Retry(waiter.AdminAccountNotFoundTimeout, func() *resource.RetryError {
+		_, err := conn.DisableSecurityHub(&securityhub.DisableSecurityHubInput{})
+
+		if tfawserr.ErrMessageContains(err, securityhub.ErrCodeInvalidInputException, "Cannot disable Security Hub on the Security Hub administrator") {
```

Comment:
added to account for error when the resource isn't set as dependency in a config